src/zoid/buttons/component.jsx

```diff
@@ -67,13 +67,13 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 fundingSource,
                 onShippingChange,
                 style = {},
-                applePaySupport = isApplePaySupported(),
                 fundingEligibility = getRefinedFundingEligibility(),
                 supportsPopups = userAgentSupportsPopups(),
                 supportedNativeBrowser = isSupportedNativeBrowser(),
                 experiment = getVenmoExperiment(enableVenmoExperiment)
             } = props;
-
+            
+            const applePaySupport = fundingEligibility && fundingEligibility[FUNDING.APPLEPAY]?.eligible ? isApplePaySupported() : false;
```

Comment:
I think you can drop the falsy check. Optional chaining does not blow up when doing `undefined.?`. Ex:

<img width="237" alt="Screen Shot 2021-06-23 at 12 14 30 PM" src="https://user-images.githubusercontent.com/534034/123140232-abb6f880-d41c-11eb-8884-2cf2fd8b9f54.png">


```suggestion
            const applePaySupport = fundingEligibility[FUNDING.APPLEPAY]?.eligible ? isApplePaySupported() : false;
```

Reply:
When I do that I get lint error.  That's why I added it.

Reply:
ahh nevermind then.

Reply:
Using now in funding config.